tests/unit/modules/test_github_repo_return.py

```diff
@@ -0,0 +1,342 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
General comment about this tests:

Where does the test code call the code in plugins/modules/repository_information.py

Reply:
@Khounborinn Please address this issue.

Reply:
According to common Python posts on StackOverflow, Future and meta class are for legacy versions of Python. It's for earlier Python versions to work with newer versions of code. Typically it's for people using Python 2 that want to use some functionality of Python 3 versions.